### PR TITLE
Brings upstream cargo features in line with our cargo changes

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
This adjusts prices and fixes a lot of oversights in my original cargo PRs, and newly added things from upstream, helping to alleviate some of the cargo powergame.

image

## Why It's Good For The Game
Cargo has been the premier site for weapons and armor, and while I don't want to take away from that, some things like medical supplies take away from other jobs, and are so cheap as to be widely available. The paramedics want their work back.

This will restore the balance

## Proof Of Testing
image
image

## Changelog

:cl: ReturnToZender
del: Removes the Lanca and Sol LMG completely from cargo orders, leaves the Sakhno to the mosin order
del: Re-removes the ion carbine from imports (it's in the goodies tab)
del: Removes extra ammunition types for no longer used weapons from import order
add: Removed company imports added to goodies and security orders
balance: adjusts prices on some company imports to not be essentially free, such as medical supplies and armor
/:cl:

### CODE REVIEW

**Comment by KateAnbolho** [comment]
Hey!

Personally, I'd reccomend looking more into either nerfing the ammo entirely (Mostly the fact that you can print .310 PHASIC from an ammo lathe), along with instead of nuking the Lancia entirely maybe moving it to the emag list for something cheaper than the Condor crate instead of axing a cool weapon entirely.

**Comment by SovietSpaghetti** [comment]
Not sure if I'm reading this right, but it looks like a lot more than just the lanca got removed? I don't see where the Bobr got moved to. Also, an 8 to 32 and a 3.5 to 16 for the synth medkits seems a lot more than just a doubled price. Considering synths basically never have any medical options other than what they can order from cargo, this feels REALLY excessive. How many times have you seen roboticists or chemists actually making this stuff? Not counting that half the stuff in the kits can't even be made stationside.

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 1]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/combat_surgeon
 	item_type = /obj/item/storage/medkit/combat_surgeon/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/robo_repair
 	item_type = /obj/item/storage/medkit/robotic_repair/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 16 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:24]:
```suggestion
	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 2]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/robo_repair_super
 	item_type = /obj/item/storage/medkit/robotic_repair/preemo/stocked
-	cost = PAYCHECK_COMMAND * 8
+	cost = PAYCHECK_COMMAND * 32 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:28]:
```suggestion
	cost = PAYCHECK_COMMAND * 16 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
Robotics and synths already has it very rough. I'm fine with double, but 4x the price is too much.

**Comment by lessthnthree** [comment]
Medical ones need adjustments in two places for it to be consistent, I will add comments.

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 3]
```diff
@@ -9,35 +9,35 @@
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/civil_defense
 	item_type = /obj/item/storage/medkit/civil_defense/stocked
-	cost = PAYCHECK_COMMAND * 2.5
+	cost = PAYCHECK_COMMAND * 5 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:12]: /datum/supply_pack/medical/civil_defense needs an adjustment to
cost = CARGO_CRATE_VALUE * 10

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 4]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/frontier
 	item_type = /obj/item/storage/medkit/frontier/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:16]: /datum/supply_pack/medical/frontier_first_aid
cost = CARGO_CRATE_VALUE * 21

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm (hunk 1 above)
Comment (lessthnthree) [reply modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:24]: And assuming it's COMMAND * 7

/datum/supply_pack/science/synth_medkits
adjustment to
cost =  CARGO_CRATE_VALUE * 14

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 5]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/robo_repair_super
 	item_type = /obj/item/storage/medkit/robotic_repair/preemo/stocked
-	cost = PAYCHECK_COMMAND * 8
+	cost = PAYCHECK_COMMAND * 32 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/first_responder
 	item_type = /obj/item/storage/backpack/duffelbag/deforest_surgical/stocked
-	cost = PAYCHECK_COMMAND * 10.5
+	cost = PAYCHECK_COMMAND * 21 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:32]: /datum/supply_pack/medical/kit_surgical and /datum/supply_pack/security/secmed_surgical adjust to
cost = CARGO_CRATE_VALUE * 10.5

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 6]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/orange_satchel
 	item_type = /obj/item/storage/backpack/duffelbag/deforest_medkit/stocked
-	cost = PAYCHECK_COMMAND * 9.5
+	cost = PAYCHECK_COMMAND * 19 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:36]: /datum/supply_pack/medical/kit_medical and /datum/supply_pack/security/secmed_medical adjust to
cost = CARGO_CRATE_VALUE * 9.5

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 7]
```diff
@@ -9,35 +9,35 @@
...
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/technician_satchel
 	item_type = /obj/item/storage/backpack/duffelbag/deforest_paramedic/stocked
-	cost = PAYCHECK_COMMAND * 11.75
+	cost = PAYCHECK_COMMAND * 23.5 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:40]: /datum/supply_pack/medical/kit_technician and /datum/supply_pack/security/secmed_technician adjust to
cost = CARGO_CRATE_VALUE * 11.75

**Comment by ReturnToZender** [comment]
I was very tired at the time and clearly fucked up a lot of math, will fix soon

**Comment by ReturnToZender** [comment]
> Not sure if I'm reading this right, but it looks like a lot more than just the lanca got removed? I don't see where the Bobr got moved to. Also, an 8 to 32 and a 3.5 to 16 for the synth medkits seems a lot more than just a doubled price. Considering synths basically never have any medical options other than what they can order from cargo, this feels REALLY excessive. How many times have you seen roboticists or chemists actually making this stuff? Not counting that half the stuff in the kits can't even be made stationside.

The Bobr is already a goodie pack. If it wasn't moved in code, it was probably already there

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm (hunk 3 above)
Comment (ReturnToZender) [reply modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:12]: Already is

**Comment by plsleavemealon** [comment]
Why up the costs of the medical kits? They are already exceptionally expensive and 9/10 its medical staff buying them....hell its usually bought _by_ parameds.  So this change hurts them the most.

**Comment by Fluffy-kitt** [comment]
Would the other guns that aren't the Lanca be possibly added to the goodies tab? the Plasma hand guns were pretty unique in being a energy gun with ammo mechanics.

**Comment by ReturnToZender** [comment]
> Would the other guns that aren't the Lanca be possibly added to the goodies tab? the Plasma hand guns were pretty unique in being a energy gun with ammo mechanics.

I did

**Comment by lessthnthree** [comment]
Before this leaves TM we can move the price changes to a modular file.

**Comment by plsleavemealon** [comment]
Why disable the ability to buy the Sakhno?

File: modular_skyrat/modules/sec_haul/code/guns/cargo_stuff.dm [hunk 8]
```diff
@@ -166,6 +167,7 @@
 		/obj/item/gun/ballistic/automatic/sol_rifle/machinegun = 1,
 		/obj/item/ammo_box/magazine/c40sol_rifle/drum = 2,
 	)
+*/ //BUBBER EDIT: IT'S AS BAD AS YOU THOUGHT
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/sec_haul/code/guns/cargo_stuff.dm:170]: Why? It's emag only already

**Comment by projectkepler-ru** [comment]
I'll move all of the gun to NRI surplus crate later down the line but good catch on some of it, the scoped gun was something I forgo about.

Also, just so you know, increasing the price on exotic shotgun ammo crate does absolutely nothing as cargo don't buy it and the only people who do is sec and time cap out at 20 mins cooldown
So I'm not sure what's the purpose of that as buying regular shotgun ammo crate for slug && buckshot is not only more economically viable they have better longterm investment return

File: modular_zubbers/code/modules/cargo/packs/security.dm [hunk 9]
```diff
@@ -42,7 +42,7 @@
 	crate_name = "security medic crate"
 	desc = "Contains a medical technician kit."
 	access = ACCESS_SECURITY
-	cost = CARGO_CRATE_VALUE * 5.5
+	cost = CARGO_CRATE_VALUE * 11.75 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```
Comment (projectkepler-ru) [inline modular_zubbers/code/modules/cargo/packs/security.dm:45]: This is only purchased as a departmental order to begin with???
Why?

File: modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 10]
```diff
@@ -0,0 +1,127 @@
+// Precompiled first aid kits, ready to go if you don't want to bother getting individual items
```
Comment (projectkepler-ru) [inline modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:1]: Ditto

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

File: modular_zubbers/code/modules/cargo/packs/security.dm (hunk 9 above)
Comment (lessthnthree) [reply modular_zubbers/code/modules/cargo/packs/security.dm:45]: It's the same one as in cargo imports.

File: modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm (hunk 10 above)
Comment (lessthnthree) [reply modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:1]: I went through the medical packs and calculated the value of items in each one. The prices where it was, it was giving out items far cheaper than medbay's own Nanomed. Medical getting a discounted version locked to themselves, and a more expensive one that's available to the general public seems fair.

Comment (lessthnthree) [reply modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:1]: For example, being able to buy a laser scalpel, mechanical pinchers, and searing tool using your roundstart cash, an hour before the research will be finished for medbay, let alone materials, is insane.

**Comment by Hemlock32** [comment]
I can't say I particularly like moving stuff out of the company imports tab, considering how much more usable that UI is compared to the main cargo UI

File: modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm (hunk 10 above)
Comment (ReturnToZender) [reply modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:1]: > For example, being able to buy a laser scalpel, mechanical pinchers, and searing tool using your roundstart cash, an hour before the research will be finished for medbay, let alone materials, is insane.
> 
> The cargo imports medical rebalance would have been my own PR, but Zender had this one so I added.

If you have a better plan for medical, I can atomize my changes to the medical items and you can take care of that, you play more medical than I do

**Comment by lessthnthree** [comment]
Yeah I was already thinking of atomising the medical changes, just haven't had the time.

Sent with Shortwave <https://www.shortwave.com?utm_medium=email&utm_content=signature&utm_source=dGhyZWVAbGVzc3RoYW50aHJlZS5zZQ==>

On Sun Oct 20, 2024, 05:16 PM GMT, Return ***@***.***> wrote:
> @ReturnToZender commented on this pull request.
------------------------------

**Comment by ReturnToZender** [comment]
Understood, I'll drop medical changes from the PR and let you take over

**Review by lessthnthree — APPROVED** [review]

File: modular_skyrat/modules/sec_haul/code/guns/cargo_stuff.dm (hunk 8 above)
Comment (ReturnToZender) [reply modular_skyrat/modules/sec_haul/code/guns/cargo_stuff.dm:170]: It's extremely powerful and is notably better than anything a traitor could possibly get, with significantly more ammo as well.

**Review by Majkl-J — CHANGES_REQUESTED** [review]
No need to leave bubber edit comments no more

**Comment by Swiftfeather** [comment]
> No need to leave bubber edit comments no more

Might as well move those files to zubbers

File: modular_skyrat/modules/company_imports/code/armament_datums/jarnsmiour.dm [hunk 11]
```diff
@@ -17,11 +17,11 @@
 
 /datum/armament_entry/company_import/blacksteel/blade/bowie_knife
 	item_type = /obj/item/storage/belt/bowie_sheath
-	cost = PAYCHECK_COMMAND * 3
+	cost = PAYCHECK_COMMAND * 6 //BUBBER EDIT: DOUBLES THE COST OF THE BOWIE AND SHAMSHIR SABRE
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/jarnsmiour.dm:20]:
```suggestion
	cost = PAYCHECK_COMMAND * 6
```

File: modular_skyrat/modules/company_imports/code/armament_datums/jarnsmiour.dm [hunk 12]
```diff
@@ -17,11 +17,11 @@
...
 
 /datum/armament_entry/company_import/blacksteel/blade/shamshir_sabre
 	item_type = /obj/item/storage/belt/sabre/cargo
-	cost = PAYCHECK_COMMAND * 3
+	cost = PAYCHECK_COMMAND * 6 //BUBBER EDIT: DOUBLES THE COST OF THE BOWIE AND SHAMSHIR SABRE
```
Comment (lessthnthree) [inline modular_skyrat/modules/company_imports/code/armament_datums/jarnsmiour.dm:24]:
```suggestion
	cost = PAYCHECK_COMMAND * 6
```

**Review by lessthnthree — COMMENTED** [review]
Let's webedit this b

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**